### ISSUE TITLE
ProgramTest.java:48-50: Object parser is missing. We need to imp...

### ISSUE THREAD

**Issue by 0pdd** [yegor256/eo#95]
The puzzle `70-9954894f` in `eo-compiler/src/test/java/org/eolang/compiler/ProgramTest.java` (lines 48-50) has to be resolved: Object parser is missing. We need to implement it, but before we should decide how to generate java code from EO object.

The [puzzle](http://www.yegor256.com/2009/03/04/pdd.html) was created by g4s8 on 20-Dec-16. 

Estimate: 30 minutes, role: IMP.

If you have any technical questions, don't ask me, submit new tickets instead. The task will be "done" when the problem is fixed and the text of the puzzle is removed from the source code.

**Comment by g4s8** [comment]
I think we can compile EO object to java class with object instance as static variable. Also we need to implement EO type compiled to java interface.

Example:
```
object file("/dev/null") as File:
    Text @path
    ctor(Text path)
    Bytes content():
        bytes:
            stream:
                @path
```
compiles into:
```java
public final class file implements File {

    public static final file OBJECT = new file("/dev/null");

    private final Text path;

    public File(final Text path) {
        this.path = path;
    }

    public Bytes content() {
        return new bytes(new stream(this.path));
    }
}
```

**Comment by yegor256** [comment]
@g4s8 looks logical

**Comment by 0pdd** [comment]
The puzzle `70-9954894f` has disappeared from the source code, that's why I closed this issue. //cc @yegor256

**Comment by yegor256** [comment]
@g4s8 many thanks!

**Comment by 0pdd** [comment]
@0pdd all 5 puzzles are solved here: [#104](https://github.com/yegor256/eo/issues/104), [#105](https://github.com/yegor256/eo/issues/105), [#106](https://github.com/yegor256/eo/issues/106), [#129](https://github.com/yegor256/eo/issues/129), [#131](https://github.com/yegor256/eo/issues/131).
